scripts/record_deployments.py

```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/env python
```

Comment:
Do we have Python3 available?

Reply:
We do on the jenkins server yeah

Reply:
Is it the default `python` ?

Reply:
Changed this to use `python3` in 4774d8550bf367e8bd868906e38e1f6c62aa7ccc